spec/03-plugins/24-response-rate-limiting/04-access_spec.lua

```diff
@@ -9,20 +9,64 @@ local REDIS_SSL_SNI   = helpers.redis_ssl_sni
 local REDIS_PASSWORD  = ""
 local REDIS_DATABASE  = 1
 
-local SLEEP_TIME = 0.01
-local ITERATIONS = 10
+local ITERATIONS = 6
```

Comment:
We don't need so many iterations to verify rate limiting in a short period(1 secs), a higher iteration increases the risk of flaky(the limiting period end and start a new period).
A smaller iterations number is enough to verify the correctness.